static/src/javascripts/bootstraps/commercial.js

```diff
@@ -35,11 +37,14 @@ define([
                 // load tags
                 robust('cm-thirdPartyTags',            function () { thirdPartyTags.init(); });
                 robust('cm-articleAsideAdverts',       function () { articleAsideAdverts.init(); });
-                robust('cm-articleBodyAdverts',        function () { articleBodyAdverts.init(); });
+                robust('cm-articleBodyAdverts',        function () {
+                    Promise.race([articleBodyAdverts.init()]).then(function () {
+                        robust('cm-dfp', function () { dfp.init(); });
+                    });
+                });
```

Comment:
I think that dfp.init should be fired after all of those methods are called, not only articleBodyAdverts